src/components/calcite-switch/calcite-switch.scss

```diff
@@ -1,88 +1,88 @@
 // theme variables
 // light theme
 :host {
-  --calcite-switch-hover-handle-border: var(--calcite-ui-brand);
-  --calcite-switch-switched-track-background: var(--calcite-ui-brand);
-  --calcite-switch-switched-track-border: var(--calcite-ui-brand);
-  --calcite-switch-switched-handle-border: var(--calcite-ui-brand);
-  --calcite-switch-switched-hover-track-background: var(--calcite-ui-brand);
-  --calcite-switch-switched-hover-track-border: var(--calcite-ui-brand);
-  --calcite-switch-switched-hover-handle-border: var(--calcite-ui-brand-hover);
+  --calcite-switch-hover-handle-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-track-background: theme("backgroundColor.brand");
+  --calcite-switch-switched-track-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-handle-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-track-background: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-track-border: theme("backgroundColor.brand");
+  --calcite-switch-switched-hover-handle-border: theme("backgroundColor.brand-hover");
 }
 
 // sizes
 :host([scale="s"]) {
-  --calcite-switch-track-width: 28px;
-  --calcite-switch-track-height: 16px;
-  --calcite-switch-handle-size: 14px;
+  --calcite-switch-track-width: theme("spacing.6");
+  --calcite-switch-track-height: theme("spacing.4");
+  --calcite-switch-handle-size: theme("spacing.3");
 }
 
 :host([scale="m"]) {
   --calcite-switch-track-width: 36px;
-  --calcite-switch-track-height: 20px;
+  --calcite-switch-track-height: theme("spacing.5");
   --calcite-switch-handle-size: 18px;
```

Comment:
@bstifle can we adjust these to land on the scale?

Reply:
@bstifle ^

Reply:
not sure what i am reviewing here

Reply:
I am referring to the track width and handle size.

Reply:
Fixed!